addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -243,7 +244,21 @@ export class PivotCorePlugin extends OdooCorePlugin {
             }
         }
 
-        this._addBorders(sheetId, position, table);
+        const pivotZone = {
+            top: position.row,
+            bottom: position.row + pivotCells[0].length - 1,
+            left: position.col,
+            right: position.col + pivotCells.length - 1,
+        };
+        const numberOfHeaders = table.getColHeaders().length - 1; // -1 to not include measures in the table headers
```

Comment:
```suggestion
        const numberOfHeaders = table.getColHeaders().length;
```
We looked at it withvincent, we think the meaasure should be part of the header (as it is a pivot header from a formula POV)

Reply:
we're going to change our mind every 2 weeks ? :smile:  (see task pad and chatter)

It's technically part of the headers yes, but it's way less (not at all) important that they stand out as the dimensions does. It's the same measure repeated, over and over again. The color and bold etc. catches the eyes but it really shouldn't.

It's certainly not at the same level of importance as the group headers.